static/specification.html

```diff
@@ -59,384 +58,130 @@
       <p>
-        Web Monetization is an API that allows websites to request small
-        payments from users facilitated by the browser and the user's Web
-        Monetization provider.
+        Web Monetization is an API that allows websites to automatically
+        receive micro-payments from users, facilitated by the browser and a
+        user's Web Monetization provider.
       </p>
     </section>
     <section id="sotd">
       <div class="warning">
         <p>
           This specification is a work in progress within the community on the
-          best shape it should take. Please see the
-          <a href="./docs/explainer">explainer</a> for more info.
+          best shape it should take. Please see the <a href=
+          "./docs/explainer">explainer</a> for more info.
         </p>
         <p>
-          The specification reflects the desired end-state of the Web Monetization
-          APIs as currently anticipated and agreed to between the contributors.
-          The specification is being prepared here, in this format, to collect the
-          input of the Web community and prepare the work to ultimately follow the
-          W3C standards track should it have the necessary support to do so.
+          The specification reflects the desired end-state of the Web
+          Monetization APIs as currently anticipated and agreed to between the
+          contributors. The specification is being prepared here, in this
+          format, to collect the input of the Web community and prepare the
+          work to ultimately follow the W3C standards track should it have the
+          necessary support to do so.
         </p>
         <p>
-          For the most accurate reflection of the APIs that have been implemented
-          by providers see the <a href="./docs/api">API documentation</a>.
+          For the most accurate reflection of the APIs that have been
+          implemented by providers see the <a href="./docs/api">API
+          documentation</a>.
         </p>
       </div>
     </section>
     <section id="terminology">
-      <h2>Terminology</h2>
+      <h2>
+        Terminology
+      </h2>
       <dl>
-        <dt><dfn>User</dfn>:</dt>
-        <dd>the party who is accessing the web-monetized website.</dd>
-        <dt><dfn>Website</dfn>:</dt>
-        <dd>the web-monetized website.</dd>
-        <dt><dfn data-lt="provider">Web Monetization provider</dfn>:</dt>
-        <dd>the party making payments on behalf of the user.</dd>
-        <dt><dfn>Web Monetization receiver</dfn>:</dt>
-        <dd>the party receiving payments on behalf of the website.</dd>
-        <dt><dfn>User agent</dfn>:</dt>
-        <dd>the web browser used by the user.</dd>
+        <dt>
+          <dfn data-lt="provider">Web Monetization provider</dfn>:
+        </dt>
+        <dd>
+          The party making payments on behalf of the user. A Web Monetization
+          provider can, for example, make use of the Simple Payment Setup
```

Comment:
This wording should be changed because making use of SPSP on top of ILP is mandated by the spec rather than being one option of several